apps/testingaccessibility/src/pages/thanks/purchase.tsx

```diff
@@ -65,30 +116,48 @@ const ThanksVerify: React.FC<React.PropsWithChildren<ThanksProps>> = ({
           />
           <div>
             <h1 className="text-orange-200 font-heading text-xl font-medium">
-              Thank you for purchasing Testing Accessibility!
+              Thank you for purchasing{' '}
+              {purchaseType === EXISTING_BULK_COUPON && 'more seats for'}{' '}
+              Testing Accessibility!
             </h1>
             {isTeamPurchase && (
               <p className="text-sand-100 max-w-md font-medium leading-relaxed mx-auto">
-                Your purchase is for <strong>{seatsPurchased}</strong> seats.
-                You can always add more seats later when your team grows.
+                Your purchase is for <strong>{seatsPurchased}</strong>{' '}
+                {purchaseType === EXISTING_BULK_COUPON && 'additional'} seats.
+                {purchaseType === NEW_BULK_COUPON &&
+                  'You can always add more seats later when your team grows.'}
               </p>
```

Comment:
I took some liberty fine-tuning the copy based on purchase type. Someone with stronger opinions should feel free to take another pass on it.

I'm also interested to hear opinions on whether inlining a bunch of conditions is sufficient here or if we should break out separate components and/or blocks of text.